CHANGES.md

```diff
@@ -63,8 +63,14 @@
   *  Add Misha to creators
   *  Add birdhouse community
 
+- Add Magpie webhook definitions for permission creation and deletion cases to be processed by Cowbird.
+- Add `USER_WORKSPACE_UID` and `USER_WORKSPACE_GID` env variables to manage ownership of the user workspaces used by
+  Cowbird, JupyterHub and others.
```

Comment:
Need to add Magpie version update from 3.31.0 -> 3.34.0. 
See other entries in the changes for the format pointing as release tags.